### ISSUE TITLE
Issue with aggregated table

### ISSUE THREAD

**Issue by alifbe** [equinor/fmu-dataio#252]
AggregatedData seems to be not ready to handle table data. There are two issues that I encountered when I tried to do that:
1. I got the error message that: `The 'name' entry is missing for constructing a file name`
https://github.com/equinor/fmu-dataio/blob/423628941dba731396f8ff4952e6c6f68cc65539/src/fmu/dataio/_filedata_provider.py#L120-L121
This seems to be originating from generating metadata using fakeconfig. 
https://github.com/equinor/fmu-dataio/blob/423628941dba731396f8ff4952e6c6f68cc65539/src/fmu/dataio/dataio.py#L1196-L1197

It works fine with test case because for some reason, aggregated surface object has a name in it (the name is "unknown"). From what I can see, this name is not being used in this case, so perhaps the easiest way to fix it is to assign a dummy name or we can just assign the name from dataio object.
`        etemp = ExportData(config=fakeconfig, name=self.name)
`

2. It assumed that there is entry `bbox` in the `data`
https://github.com/equinor/fmu-dataio/blob/423628941dba731396f8ff4952e6c6f68cc65539/src/fmu/dataio/dataio.py#L1210
We can probably fix it by checking for the existence of `bbox`
```
        if etempmeta["data"].get("bbox"):
            template["data"]["bbox"] = etempmeta["data"]["bbox"]
```

3. Having fixed those issues and got the aggregated metadata, I see that the parameters is not transferred to the aggregated metadata. It could be interesting to see the list of parameters that is input to the aggregated data. From the webviz side, it's probably easier for webviz to extract this information and use it for the various plugins that they already have. Or perhaps there are other reasons for not transfer those information?

**Comment by perolavsvendsen** [comment]
Ref 3), including parameters on aggregated data. This is not straight forward, and needs a bit of thinking and discussion. I am unsure how to actually do this properly, as the parameters will be different for all the input data objects. So, the parameters for an _aggregated_ object is really the distribution of each parameter from the source objects.

**Comment by perolavsvendsen** [comment]
Ref 1), I think this should trigger the user to actually enter a name in the call to dataio. Perhaps the error message could be improved a bit.

**Comment by alifbe** [comment]
> Ref 1), I think this should trigger the user to actually enter a name in the call to dataio. Perhaps the error message could be improved a bit.

But from what I can see, the name is not being used anywhere (only `file `and `tracklog` field is extracted from generated `etempmeta`) So, it is probably fine to just assign any arbitrary name

https://github.com/equinor/fmu-dataio/blob/423628941dba731396f8ff4952e6c6f68cc65539/src/fmu/dataio/dataio.py#L1196-L1212

**Comment by alifbe** [comment]
> Ref 3), including parameters on aggregated data. This is not straight forward, and needs a bit of thinking and discussion. I am unsure how to actually do this properly, as the parameters will be different for all the input data objects. So, the parameters for an _aggregated_ object is really the distribution of each parameter from the source objects.

Exactly. So, I was thinking that the parameters in the aggregated class object could be the list of the source object parameters. This will be analog to aggregating parameters.txt in the scratch disc. The drawback is that the metadata could potentially become too big.

**Comment by alifbe** [comment]
Fixed with PR #254
